lib/gpu_memory_service/snapshot/backends/pinned_host.py

```
@@ -0,0 +1,109 @@
+
+def make_pinned_copy_slots(count: int) -> List[PinnedCopySlot]:
+    slots: List[PinnedCopySlot] = []
+    try:
+        for _ in range(count):
+            slots.append(PinnedCopySlot())
+    except Exception:
+        for slot in slots:
+            try:
+                slot.close()
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)